### PR TITLE
Improve metadata for social media preview cards

### PR DESCRIPTION
**Pull request by NickCrews**
- Add missing og:url and og:type fields
- Localize description field if a city link is shared.

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -20,8 +20,14 @@
     <!-- Facebook Open Graph protocol -->
     <meta property="og:title" content="{{ site.title }}" />
     <meta property="og:site_name" content="{{ site.title }}" />
-    <meta property="og:description" content="{{ site.meta }}" />
     <meta property="og:image" content="{{ prepend: site.baseurl | prepend: site.url }}/images/defund12.org.png">
+    <meta property="og:url" content="{{ site.baseurl }}">
+    <meta property="og:type" content="website">
+    {% if page.title %}
```
Comment (NickCrews) [inline _includes/header.html:26]: I'm not sure I like how it's not immediately obvious how `page.title` is used within `site.meta_localized`. I was thinking of doing something more like

```
meta: Send pre-populated localized emails to officials in

{% if page.title %}
 <meta property="og:description" content="{{ site.meta }} {{ page.title }}." />
  {% else %}
  <meta property="og:description" content="{{ site.meta }} your city." />
  {% endif %}
```
but that's kinda gross too...

File: _includes/header.html [hunk 2]
```diff
@@ -14,20 +14,30 @@
...
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="{{ site.baseurl }}">
 
     <!-- Twitter card protocol -->
     <meta name="twitter:card" content="summary_large_image">
     <meta name="twitter:title" content="{{ site.title }}">
     <meta name="twitter:image" content="{{ prepend: site.baseurl | prepend: site.url }}/images/defund12.org.png">
+
+    <!-- Localized descriptions for all the above protocols -->
+    {% if page.title %}
```
Comment (NickCrews) [inline _includes/header.html:32]: I'm not sure I like how it's not immediately obvious how page.title is used within site.meta_localized. I was thinking of doing something more like

```
meta: Send pre-populated localized emails to officials in

{% if page.title %}
 <meta property="og:description" content="{{ site.meta }} {{ page.title }}." />
  {% else %}
  <meta property="og:description" content="{{ site.meta }} your city." />
  {% endif %}
```

but that's kinda gross too...

File: _includes/header.html (hunk 1 above)
Comment (NickCrews) [reply _includes/header.html]: Moved this comment to the newer version

**Comment by NickCrews** [comment]
per @Ceshion let's at least wait post-migration to a framework before adding this

File: _config.yml [hunk 3]
```diff
@@ -29,8 +29,9 @@ permalink: pretty
 subtitle: Email government officials and council members to reallocate egregious police
   budgets towards education, social services, and dismantling racial injustice via
   this crowd-sourced tool.
-meta: Email government officials and council members to reallocate police budgets
-  towards education, social services, and dismantling racial injustice.
+# On Facebook the preview card only shows the first ~75 chars, so keep it short
+meta: Send pre-populated localized emails to officials in your city.
+meta_localized: Send pre-populated localized emails to officials in {{ page.title }}.
```
Comment (skullface) [inline _config.yml:34]: Ooooh great catch on the short character limit for FB!

“Send pre-populated localized emails to officials” feels a little wordy/overly technical for a mass audience. Something more like this?

```suggestion
  meta: Send a pre-written email with your city’s specific budget concerns.
  meta_localized: Send a pre-written email to directly to {{ page.title }} officials.
```

Simplified from Grade 10 to Grade 8 [via Hemingway](http://www.hemingwayapp.com/)~

**Comment by skullface** [comment]
We can also add additional context to the actual page title~

https://github.com/defund12/defund12.org/pull/1525/files#diff-822e9cf54ee310f9a7ebd0d16c9ae066R7

```diff
-  <title>{{ site.title }}</title>
+  <title>{% if page.title %}{{ page.title | escape }} on Defund12.org{% else %}{{ site.title }}{% endif %}</title>
```

File: _config.yml (hunk 3 above)
Comment (NickCrews) [reply _config.yml]: Nice! Better.

**Comment by NickCrews** [comment]
Just did two small fixes. Got frustrated trying to learn react and Gatsby. Someone else take over if you want!

**Comment by NickCrews** [comment]
@bingbongle @Ceshion There is still two small tweaks here that are probably good.

**Comment by bingbongle** [comment]
It looks like this was added via: https://github.com/defund12/defund12.org/pull/1596

**Pull request closed without merging**